doc/modules/ROOT/pages/python-runtime/knowledge-graph-embeddings.adoc

```diff
@@ -0,0 +1,283 @@
+All scheduler parameters except `optimizer` can be passed as `lr_scheduler_kwargs`.
+
+=== Inner normalisation
+
+In the original `TransE` paperfootnote:[Bordes, Antoine, et al. "Translating embeddings for modeling multi-relational data." Advances in neural information processing systems 26 (2013).]
+in `Algorithm 1`, line 5, the entity embeddings are normalized to have `Lp` norm of 1.
+For some datasets, this normalization might not be beneficial.
+To avoid this normalization, set `inner_norm` to `False`.
+
+
+[[algorithms-embeddings-kge-syntax]]
+== Syntax
+
+[source,python]
+----
+gds.model.transe.train(G,
+    num_epochs = 10,
+)
+----
+
+.Parameters
+[cols="1m,1m,1m,1", options="header"]
+|====
+| Parameter | Type | Default value | Description
+
+| num_epochs
+| int
+| N/A
+| Number of epochs for training (must be greater than 0)
+
+| embedding_dimension
+| int
+| 256
+| Dimensionality of the embeddings (must be greater than 0)
+
+| epochs_per_checkpoint
+| int
+| max(num_epochs / 10, 1)
+| Number of epochs between checkpoints (must be greater than or equal to 0)
+
+| load_from_checkpoint
+| Optional[tuple[str, int]]
+| None
+| Checkpoint to load from, specified as a tuple (path, epoch)
+
+| split_ratios
+| dict[str, float]
+| {TRAIN=0.8, TEST=0.2}
+| Ratios for splitting the dataset into training and test sets
+
+| scoring_function
+| str
+| "transe"
+| Function used to score embeddings
+
+| p_norm
+| float
+| 1.0
+| Norm to use in TransE scoring function
+
+| batch_size
+| int
+| 512
+| Size of the training batch (must be greater than 0)
+
+| test_batch_size
+| int
+| 512
+| Size of the test batch (must be greater than 0)
+
+| optimizer
+| str
+| "adam"
+| Optimizer to use for training
+
+| optimizer_kwargs
+| dict[str, Any]
+| {lr=0.01, weight_decay=0.0005}
+| Additional arguments for the optimizer
+
+| lr_scheduler
+| str
+| ConstantLR
+| Learning rate scheduler
+
+| lr_scheduler_kwargs
+| dict[str, Any]
+| {factor=1, total_iters=1000}
+| Additional arguments for the learning rate scheduler
+
+| loss_function
+| str
+| MarginRanking
+| Loss function to use for training
+
+| loss_function_kwargs
+| dict[str, Any]
+| {margin=1.0, adversarial_temperature=1.0, gamma=20.0}
+| Additional arguments for the loss function
+
+| negative_sampling_size
+| int
+| 1
+| Number of negative samples per positive sample
+
+| use_node_type_aware_sampler
+| bool
+| False
+| Whether to sample negative nodes with the same label as the corresponding positive node
+
+| k_value
+| int
+| 10
+| Value of k used in evaluation metrics
+
+| do_validation
+| bool
+| True
+| Whether to perform validation
+
+| do_test
+| bool
+| True
+| Whether to perform testing
+
+| filtered_metrics
+| bool
+| False
+| Whether to use filtered metrics during evaluation
+
+| epochs_per_val
+| int
+| 50
+| Number of epochs between validations (must be greater than or equal to 0)
+
+| disable_tqdm
+| bool
+| True
+| Whether to disable tqdm progress bars
```

Comment:
TQDM is not something that is user facing right? I think what the user will see is something like MLFlow?